src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -29,11 +30,26 @@ public void promptNextPlayer(){
         }
     }
 
+    public void printScore(){
```

Comment:
This function does not need to be public. It can be private since we only use it in the TicTacToeGame class.